features/enable_fips_vm.feature

```diff
@@ -5,26 +5,46 @@ Feature: FIPS enablement in lxd VMs
   Scenario Outline: Attached enable of FIPS in an ubuntu lxd vm
     Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
     When I attach `contract_token` with sudo
-    When I run `pro status --format json` with sudo
-    Then stdout contains substring
+    And I apt install `jq`
+    And I run shell command `pro status --format json` with sudo
+    And I apply this jq filter `'.services[] | select(.name == "livepatch") | .available'` to the output
+    Then I will see the following on stdout
       """
-      {"available": "yes", "blocked_by": [{"name": "livepatch", "reason": "Livepatch cannot be enabled while running the official FIPS certified kernel. If you would like a FIPS compliant kernel with additional bug fixes and security updates, you can use the FIPS Updates service with Livepatch.", "reason_code": "livepatch-invalidates-fips"}], "description": "NIST-certified FIPS crypto packages", "description_override": null, "entitled": "yes", "name": "fips", "status": "disabled", "status_details": "FIPS is not configured", "warning": null}
+      "yes"
```

Comment:
sorry I do think we can have a second check here for the FIPS blocked_by field. I know we may test the functionality/interaction between FIPS/Livepatch later in the flow, but here we want to assure we show the blocked_by reason in the API imho